.github/workflows/odh-kuberay-client-release.yaml

```
@@ -0,0 +1,46 @@
+      - dev
+      - release-*
+    paths:
+      - 'clients/python-client/**'
+      - '.github/workflows/odh-kuberay-client-release.yaml'
+
+jobs:
+  build-and-publish:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)